egs/iban/s5/cmd.sh

```diff
@@ -0,0 +1,3 @@
+export train_cmd="run.pl --max-jobs-run 32"
+export decode_cmd="run.pl --max-jobs-run 32"
+export cuda_cmd="run.pl --max-jobs-run 4"
```

Comment:
We shouldn't be using cuda_cmd in new recipes at all.  That was an nnet1
thing.

Dan


On Fri, Apr 22, 2016 at 4:52 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/iban/s5/cmd.sh
> <https://github.com/kaldi-asr/kaldi/pull/727#discussion_r60808087>:
>
> > @@ -0,0 +1,3 @@
> > +export train_cmd="run.pl --max-jobs-run 32"
> > +export decode_cmd="run.pl --max-jobs-run 32"
> > +export cuda_cmd="run.pl --max-jobs-run 4"
>
> I don't think the AWS g2 instances have multiple GPUs per node.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/727/files/eedfdca7e9a3e78ba82aa49328bae01a46f09cd9#r60808087>
>